r/src/compute.cpp

```diff
@@ -188,6 +188,15 @@ std::shared_ptr<arrow::compute::FunctionOptions> make_compute_options(
     return out;
   }
 
+  if (func_name == "one" || func_name == "hash_one") {
+    // (TODO) FunctionOption for `hash_one` has not been implemented yet, so
```

Comment:
Does it need options? Do we have other cases where we are calling functions without options (surely we do), and how are those handled?

Reply:
I forgot to remove the old comments, and there is no longer any reason to separate the blocks, so I put them together with other Aggreate functions.